### PR TITLE
Add Bootstrap breadcrumbs for main, projects and packages

### PR DESCRIPTION
**Pull request by eduardoj**
Co-authored-by: Moisés Déniz Alemán <mdeniz@suse.com>

Screenshot of a package page:
package_breadcrumbs

### CODE REVIEW

**Comment by eduardoj** [comment]
[P6: Implement breadcrumbs](https://trello.com/c/xlyYssdo/838-p6-implement-breadcrumbs)

File: src/api/app/views/layouts/webui2/_breadcrumbs.html.haml [hunk 1]
```diff
@@ -1,10 +1,3 @@
...
   %ol.breadcrumb.bg-transparent
-    - @breadcrumbs.each do |breadcrumb|
-      %li.breadcrumb-item{ class: @breadcrumbs.last == breadcrumb && 'active' , "aria-current" => "page" }
-        - if breadcrumb == @breadcrumbs.first
-          %i.fas.fa-home
-        - if breadcrumb[:path]
-          = link_to breadcrumb[:name], breadcrumb[:path]
-        - else
-          = breadcrumb[:name]
+    = render partial: 'breadcrumb_items'
```
Comment (hennevogel) [inline src/api/app/views/layouts/webui2/_breadcrumbs.html.haml]: nitpick: why dont you move this partial on here? doesn't make much sense to have them seperated IMHO

File: src/api/app/views/webui2/webui/package/_breadcrumb_items.html.haml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+= render partial: 'webui/project/breadcrumb_items'
+%li.breadcrumb-item
+  = link_to @package, package_show_path(@package)
```
Comment (hennevogel) [inline src/api/app/views/webui2/webui/package/_breadcrumb_items.html.haml]: This is missing the project and `link_to @object` is a link already so this becomes `/package/show/$package/$package`

**Comment by dmarcoux** [comment]
Do we need to update the `breadcrumbs` component in [OBS patterns](https://obs-patterns.netlify.com/components/breadcrumbs.html)?

**Comment by hennevogel** [comment]
> Do we need to update the breadcrumbs component in OBS patterns?

Yes of course

**Comment by mdeniz** [comment]
>    Do we need to update the breadcrumbs component in OBS patterns?

> Yes of course

In which it have changed? The HAML stays the same...

File: src/api/app/views/webui2/webui/package/_breadcrumb_items.html.haml (hunk 2 above)
Comment (eduardoj) [reply src/api/app/views/webui2/webui/package/_breadcrumb_items.html.haml]: Done.

**Comment by hennevogel** [comment]
> In which way have the pattern changed? The HAML stays the same...

True that

**Pull request merged**